### ISSUE TITLE
Should `text-decoration-thickness: auto` and `from-font` round to device pixels and ensure >= 1px?

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#8768]
https://drafts.csswg.org/css-text-decor-4/#text-decoration-thickness-property says this for lengths and percentages:

> The UA should round the actual value to the nearest integer device pixel, and ensure it is at least one device pixel.

However, I don't see anything similar for `auto` nor `from-font`.

In servo this means that the thickness can get below 1px and not be painted: https://github.com/servo/servo/issues/29668

I think it would be good to do that for all values, not just lengths and percentages.

L3 doesn't have `text-decoration-thickness` and leaves it up to the UA, but could also round to >= 1device pixels.

**Comment by Loirooriol** [comment]
Also, other than rounding thickness to device pixels, should it also be positioned in device pixels, so that it properly lines up with the pixel grid?

**Comment by emilio** [comment]
I think we just resolved this in https://github.com/w3c/csswg-drafts/issues/12696

**Comment by Loirooriol** [comment]
Yes, @fantasai reopen and add your Agenda+ back if there was something missing.
